package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "scripts": {
+    "build": "npx rollup -c"
```

Comment:
You may want to call just rollup without npx here to actually use the installed package

Reply:
Forgot about that!